Updates for `Sumida` protocol upgrade (#147)
autonity/docs.autonity.org: node-operators/install-aut/index.md

A public-facing internet connection with static IP is required:

* If you are using a cloud provider for node hosting, then a static IP address for your cloud space should be a stated hosting requirement. Cloud vendors typically don't supply a static IP address unless it is purchased explicitly.

::: {.callout-note title="Why is a static IP Address required?" collapse="false"}
Running an Autonity node requires that you maintain a static ip address because the ip address forms part of the node's network address, the [enode URL](/glossary/#enode). The enode provides the network location of the node client for p2p networking. Changing the ip address will change the node's identity on the network, therefore. This is significant if you are operating a validator node as the enode is provided as part of the validator meta data when [registering a validator](/validators/register-vali/#step-3.-submit-the-registration-transaction).
:::

Incoming traffic must be allowed on the following:

* `TCP, UDP 30303` for node p2p (DEVp2p) communication for transaction gossiping.

You may also choose to allow traffic on the following ports:

* `TCP 8545` to make http RPC connections to the node.

> * `TCP 8546` to make WebSocket RPC connections to the node (required if you are operating a validator node).

Reviewer: Same thing here, opening traffic on 8546 is actually not mandatory (unless you run the oracle on a separate machine) and must be done with caution.

A public-facing internet connection with static IP is required:

* If you are using a cloud provider for node hosting, then a static IP address for your cloud space should be a stated hosting requirement. Cloud vendors typically don't supply a static IP address unless it is purchased explicitly.

::: {.callout-note title="Why is a static IP Address required?" collapse="false"}
Running an Autonity node requires that you maintain a static ip address because the ip address forms part of the node's network address, the [enode URL](/glossary/#enode). The enode provides the network location of the node client for p2p networking. Changing the ip address will change the node's identity on the network, therefore. This is significant if you are operating a validator node as the enode is provided as part of the validator meta data when [registering a validator](/validators/register-vali/#step-3.-submit-the-registration-transaction).
:::

Incoming traffic must be allowed on the following:

* `TCP, UDP 30303` for node p2p (DEVp2p) communication for transaction gossiping.

You may also choose to allow traffic on the following ports:

* `TCP 8545` to make http RPC connections to the node.

> * `TCP 8546` to make WebSocket RPC connections to the node (for example, if you are operating as a validator node to allow a connection from your oracle server if it is running on a different host).

Reviewer: ```suggestion
* `TCP 8546` to make WebSocket RPC connections to the node (for example, if you are operating a validator node and your oracle server is hosted on a separate dedicated machine).
```
